charts/camunda-platform/templates/_helpers.tpl

```diff
@@ -49,3 +49,17 @@ Set imagePullSecrets according the values of global, subchart, or empty.
 []
 {{- end -}}
 {{- end -}}
+
+{{/*
+Keycloak service name should be a max of 20 char since the keycloak bitnami Chart is using websphere.
+Also, we needed a way to change the keycloak fullname inside the subchart config, which is possible only by using
+the global config.
```

Comment:
```suggestion
Keycloak service name should be a max of 20 char since the Keycloak Bitnami Chart is using Wildfly, the node identifier in WildFly is limited to 23 characters.

Furthermore, this allows changing the referenced Keycloak name inside the sub-charts. Subcharts can't access values from other sub-charts or the parent, global only. This is the reason why we have a glove value to specify the Keycloak full name.
```

https://github.com/bitnami/charts/blob/master/bitnami/keycloak/templates/_helpers.tpl#L2-L5